components/error-summary/src/index.js

```diff
@@ -115,29 +115,29 @@ const StyledErrorSummary = styled('div')({
 const ErrorSummary = ({
   onHandleErrorClick, heading, description, errors, ...props
 }) => (
-  <StyledErrorSummary tabIndex={-1} {...props}>
-    <Header level={2}>{ heading }</Header>
-    { description && <Paragraph mb={2}>{ description }</Paragraph> }
-    { errors.length > 0 &&
-      <UnorderedList listStyleType="none">
-        { errors.map(error => (
-          <ListItem key={error.targetName}>
-            <StyledErrorText
-              tabIndex={-1}
-              onClick={() => onHandleErrorClick(error.targetName)}
-            >
-              {error.text}
-            </StyledErrorText>
-          </ListItem>
+    <StyledErrorSummary tabIndex={-1} {...props}>
+      <Header level={3}>{heading}</Header>
```

Comment:
This is the only code change, but something has adjusted all the formatting - if you could revert the formatting changes, so the code change (from `2` - `3`) as the only change that would be mega.